### DISCUSSION TITLE
create mutation one-to-many

### DISCUSSION THREAD

**Discussion by kawemv** [prisma/prisma#11486] [Q&A]
Hi!
I'm new to programming, I started a project using PRISMA + GRAPHQL + APOLLO SERVER. I would like to create an order control system, having the following tables: CLIENT, PRODUCT and ORDER, the ORDER table derived a new table called ORDER, where an array of products is stored where the mutation should have the following return:
{
   "date": {
     "id": 1
     "order":{
       "id": 1
     }
     "product": [
       {
         "id": 1
         "name": "test"
       },
       {
         "id": 2
         "name": "test2"
       },
       {
         "id": 3
         "name": "test3"
       }
     ]
   }
}

But I couldn't understand how to make the MUTATION to connect the PEDIDOITEM table with the PRODUCT table, because when executing the mutation to create the PEDIDOITEM, an ARRAY of products must be saved. Can anyone help me?
Follow the github repository
https://github.com/kawemv/PrismaJs.git

Note: I'm using JAVASCRIPT, with POSTGRES database

**Comment by austincrim** [comment]
Hey @kawemv 👋 ,

You will want to take a look at our [nested writes documentation](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-writes). Given your current schema, a query might look something like this to create a `PedidoItem`:
```ts
await prisma.pedidoItem.create({
  data: {
    produto: {
      create: {
        ...produto data
      }
    }
  }
})
```

Let me know if I can clarify further!

**Reply by vladi-stevanovic** [reply]
👋 Hello @kawemv I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
